composer.json

```diff
@@ -22,13 +22,13 @@
         "laminas/laminas-servicemanager": "<3.0"
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-zendframework-bridge": "^1.4.0"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~1.0.0",
-        "laminas/laminas-servicemanager": "^3.0.3",
-        "phpunit/phpunit": "^9.5"
+        "laminas/laminas-coding-standard": "^2.0.0",
```

Comment:
Please stay with version 1 for the coding standard to avoid major changes to the code. This also mean to support version 7.3 of PHP.
Updating the coding standard is another task and topic.

Reply:
Version 1 is using PHP-CS v2.9.2. And it doesn't work at PHP 8.1 because it use removed feature "Array and string offset access syntax with curly braces". So, QA checks will not pass then.

Reply:
I think we can solve this by 3 different steps / pull requests:
1) Drop support for PHP 7.3 because we need laminas/laminas-servicemanager at least v3.8.0 where PHP 8.1 support added and this version requires PHP 7.4 as minimal version.
2) Update coding standard and fix related errors according to new version (just without BC changes now).
3) Update laminas/laminas-servicemanager to v3.8.0 and add support for PHP 8.1 to this package.

Reply:
@webman 
Like in other packages:

```json
{
    "ignore_php_platform_requirements": {
        "8.1": true
    }
}
```

https://github.com/laminas/laminas-hydrator/blob/7c1a95863e7f502f931f1b3a304d7e244d1239f1/.laminas-ci.json#L1-L5